.drone.yml

```diff
@@ -186,6 +186,7 @@ steps:
 services:
 - name: Start Docker
   image: docker:dind
+  privileged: true
```

Comment:
Sorry I don't have a lot of context on Drone migration but I do have a couple of questions looking at the changeset:

* Why is the privileged flag now required for all steps?
* How was it working before without privileged containers?
* Do all steps need privileged containers (seems like it got added to all)?

Reply:
Yeah, I just realised this wasn't helpful. I've added some clarity in the top PR comment.

Reply:
Thank you @webvictim, it makes sense.